src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
Class `Circle` should not extend `Figure` if `Figure` is an abstract class used to set behavior. Consider using an interface to define common behavior and use it to implement `Circle` if `Figure` is indeed abstract.